charts/registry-creds/templates/secrets_ecr.yaml

```
@@ -8,11 +8,11 @@ metadata:
         app: registry-creds
         cloud: ecr
 data:
-    AWS_ACCESS_KEY_ID: {{ .Values.ecr.awsAccessKeyId | b64enc }}
-    AWS_SECRET_ACCESS_KEY: {{ .Values.ecr.awsSecretAccessKey | b64enc }}
+    AWS_ACCESS_KEY_ID: "{{ .Values.ecr.awsAccessKeyId | b64enc }}"
```

Would you mind using the quote function instead? (and same for the other variables)

```suggestion
    AWS_ACCESS_KEY_ID: {{ .Values.ecr.awsAccessKeyId | b64enc | quote }}
```

---

done